tools/windows/DatadogAgentInstaller/WixSetup/Datadog/AgentInstaller.cs

```diff
@@ -105,8 +94,10 @@ public Project ConfigureProject()
                 new RegKey(
                     _agentFeatures.MainApplication,
                     RegistryHive.LocalMachine, @"Software\Datadog\Datadog Agent",
-                    new RegValue("InstallPath", "[PROJECTLOCATION]") { Permissions = new[] { new Permission { User = "[DDAGENTUSER_FQ_NAME]", GenericAll = true } } },
-                    new RegValue("ConfigRoot", "[APPLICATIONDATADIRECTORY]") { Permissions = new[] { new Permission { User = "[DDAGENTUSER_FQ_NAME]", GenericAll = true } } }
+                    new RegValue("InstallPath", "[PROJECTLOCATION]") { Win64 = true },
+                    new RegValue("ConfigRoot", "[APPLICATIONDATADIRECTORY]") { Win64 = true },
+                    new RegValue("installedDomain", "[DDAGENTUSER_DOMAIN]") { Win64 = true },
+                    new RegValue("installedUser", "[DDAGENTUSER_USERNAME]") { Win64 = true }
```

Comment:
Interesting inconsistencies in naming. You have changed var name from DDAGENTUSER_USER to DDAGENTUSER_USERNAME but use installedUser vs installedUserName. Nothing needs to be done just thinking aloud.

Reply:
Overall I do not see any problem. There are few potential cosmetic comments

Reply:
`DDAGENTUSER_NAME` and the `installedUser` registry value are carry overs from the current installer.

 `DDAGENTUSER_NAME` is provided by the user as `domain\user` and I didn't like how its meaning is changed by `ProcessDdAgentUserCredentials` to just contain `user` and `domain` is stored in `DDAGENTUSER_DOMAIN`. So I created `DDAGENTUSER_USERNAME` so we can leave the original user provided property `DDAGENTUSER_NAME` alone.

Reply:
Ah, I did not get this